### PR TITLE
transak integration #490

### PR DESCRIPTION
**Pull request by EduardoGomez951020**
Adding Modal for buy SKILL and transak

Screen Shot 2021-08-21 at 12 20 06

### CODE REVIEW

File: frontend/src/store.ts [hunk 1]
```diff
@@ -241,7 +241,9 @@ export function createStore(web3: Web3) {
       getExchangeUrl() {
         return 'https://app.apeswap.finance/swap?outputCurrency=0x154a9f9cbd3449ad22fdae23044319d6ef2a1fab';
       },
-
+      getExchangeTransakUrl() {
+        return 'https://global.transak.com/?apiKey=YOUR_PRODUCTION_API_KEY?defaultCryptoCurrency=BNB'; //INSERT API KEY TRANSAK
```
Comment (seiyria) [inline frontend/src/store.ts:245]: this should be marked as draft if we don't have an api key

**Review by seiyria — CHANGES_REQUESTED** [review]
needs an api key

**Comment by EduardoGomez951020** [comment]
Hi @seiyria. Where or how should I get the api key ? 

Is existing already on the .env file ? 

Screen Shot 2021-08-23 at 11 01 54

**Comment by seiyria** [comment]
@EduardoGomez951020 I have requested one from kroge, but as of right now there isn't one.

**Comment by EduardoGomez951020** [comment]
Hi @seiyria, any update about it ?

**Comment by seiyria** [comment]
looks fine, it took a while to get the api key (it looks like they forgot about us). sorry for the delay. this needs a round of testing then it would be good to go if it all works.

**Comment by bilbolPrime** [comment]
Created PR 742 with some modifications to this one. Had trouble cloning the repo.

**Pull request closed without merging**